qt/aqt/mediasrv.py

```diff
@@ -172,8 +172,13 @@ def _handle_local_file_request(request: LocalFileRequest) -> Response:
     try:
         mimetype = _mime_for_path(fullpath)
         if os.path.exists(fullpath):
+            if fullpath.endswith(".css"):
+                # make changes to css files immediately reflected in the webview
+                max_age = 2
+            else:
+                max_age = 60 * 60
```

Comment:
#### value of `max_age`
- css
  - I think `1` second should theoretically be enough for the preload to work properly, but for some reason it fails very occasionally, so I set it to `2` seconds to make sure it works.
- non-css
  - I was wondering if it would be better to change it to `None`, but I wasn't sure, so I left it at `60 * 60`.

Reply:
Perhaps it's worth making it 5 or 10 seconds, just to avoid the chance that it will still happen on users with slow computers?

For non-css, my main concern is possible redrawing when the user has embedded fonts, but it's not something I tested.

Reply:
> Perhaps it's worth making it 5 or 10 seconds, just to avoid the chance that it will still happen on users with slow computers?

Done.

> For non-css, my main concern is possible redrawing when the user has embedded fonts, but it's not something I tested.

I see. It's true that if you set `max_age` to `None` and load a font with `@font-face`, the font will be loaded every time and only the characters using that font will be rendered with some delay, which is annoying. It looks like it's best to keep the caching enabled as is.